src/llmcompressor/args/dataset_arguments.py

```diff
@@ -78,8 +78,8 @@ class CustomDatasetArguments(DVCDatasetArguments):
 @dataclass
 class DatasetArguments(CustomDatasetArguments):
```

Comment:
Other arguments we should consider removing/deprecating are
* concatenate_data
* splits
* num_calibration_samples
* pad_to_max_length
* min_tokens_per_module (separate from training, but no longer used)

Reply:
I'll take a note and do in a follow-up